backend/src/test/kotlin/org/loculus/backend/controller/submission/GetReleasedDataEndpointTest.kt

```diff
@@ -270,6 +271,21 @@ class GetReleasedDataEndpointTest(
         }
     }
 
+    @Test
+    fun `GIVEN multiple processing pipelines have submitted data THEN no duplicates are returned`() {
```

Comment:
@fhennig, should we maybe improve the test a bit further and test that the data of the right version are returned?

Reply:
Thanks! To make the test _even_ better, what do you think about checking that the result set is also correct before calling `useNewerProcessingPipelineIfPossible()`? (feel free to ignore if you've written enough tests for now!)